tools/analyse_query_usage.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage:
+#
+#   ./analyse_query_usage.sh my-file-of-urls.txt
+#
+#   cat my-file-of-urls.txt | ./analyse_query_usage.sh
+#
+#
+# For the input, a list of URLs, shows which are most common when you
+# ignore all query parameters. Eg, those that will be normalised in
+# mappings unless some query parameters are marked as important.
+#
+# Eg:
+#   162 http://www.planningportal.gov.uk/PpPortalSupport/ChangeLanguage
+#    45 http://www.planningportal.gov.uk/PpWeb/jsp/redirect.jsp
+#       ...
+#
+
+# Input is either file arg contents or stdin
+cat "$@" | \
+  # Take all of the URL leading up to `?`, or all if not present
+  cut -d? -f1 | \
```

Comment:
I thought about that, and tbh it's one of the reasons I added support fir pipable input, as I was doing it separately.

I _think_ it would be better for this script to only be concerned with normalising by query params, and if URL normalisation (which could potentially include other things) is required, then to do that to the input before this script gets it. Feels more unix-y?